examples/40_advanced/example_metrics.py

```diff
@@ -106,9 +106,13 @@ def error_wk(solution, prediction, dummy):
 cls.fit(X_train, y_train)
 
 predictions = cls.predict(X_test)
-print("Accuracy score {:g} using {:s}".
-      format(sklearn.metrics.accuracy_score(y_test, predictions),
-             cls.automl_._metric.name))
```

Comment:
Just an example of where the example was incorrect, hence me wanting to split it out and make it more clear. In general my preference is to not have any calculating functions performed in a formatted string as it can easily brushed over.
However `cls._automl_._metric.name` could definitely go into the string, there probably wasn't much point to take it out other than just to show how to get the metric name as a single expression.

Reply:
Alright.